### PR TITLE
Update Mixin dashboards

### PR DESCRIPTION
**Pull request by mcbenjemaa**
This PR

- Updates mixin dashboards from [upstream](https://github.com/kubernetes-monitoring/kubernetes-mixin)

issue: https://github.com/giantswarm/giantswarm/issues/19855

### Checklist

- [X] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,3 +39,39 @@ To upload a dashboard while editing, run:
 * grafonnet: https://github.com/grafana/grafonnet-lib
 
 `git clone https://github.com/grafana/grafonnet-lib.git $GOPATH/src/github.com/grafana/grafonnet-lib`
+
+
+
+### Mixin
```
Comment (mcbenjemaa) [inline README.md:45]: @TheoBrigitte Here is the note about the update.

Comment (TheoBrigitte) [reply README.md:45]: Perfect that's very clear, thanks <3

**Review by TheoBrigitte — APPROVED** [review]
Nice job.

**Review by hervenicol — APPROVED** [review]
Great! :+1:

**Pull request merged**